tensorflow_addons/image/transform_test.py

```diff
@@ -141,6 +143,162 @@ def test_transform_eager(self):
             np.array([[4, 4], [4, 4]]), transform_ops.transform(
                 image, [1] * 8))
 
+    @test_utils.run_in_graph_and_eager_modes
+    def test_zeros(self):
+        for dtype in _DTYPES:
+            with self.cached_session():
+                for shape in [(5, 5), (24, 24), (2, 24, 24, 3)]:
+                    for angle in [0, 1, np.pi / 2.0]:
+                        image = tf.zeros(shape, dtype)
+                        self.assertAllEqual(
+                            transform_ops.rotate(image, angle),
+                            np.zeros(shape, dtype.as_numpy_dtype()))
+
+
+class RotateOpTest(tf.test.TestCase):
```

Comment:
My bad. Can we use run_all_in_graph_and_eager_modes for RotateOpTest, and remove those decorators of test_* methods ? Note the difference: run_**all**_in_graph_and_eager_modes, see https://github.com/tensorflow/addons/tree/master/tensorflow_addons/image#testing-requirements :-)

Reply:
done